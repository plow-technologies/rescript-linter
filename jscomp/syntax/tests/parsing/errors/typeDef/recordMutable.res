type d = {
  foo: string, mutable
}
