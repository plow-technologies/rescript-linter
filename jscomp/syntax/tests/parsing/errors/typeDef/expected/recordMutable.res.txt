
  Syntax error!
  tests/parsing/errors/typeDef/recordMutable.res:2:16-3:1

  1 │ type d = {
  2 │   foo: string, mutable
  3 │ }
  4 │ 

  The `mutable` qualifier can only be used at the beginning of a field declaration

type nonrec d = {
  foo: string }